custom/icds_reports/ucr/data_sources/hardware_cases.json

```diff
@@ -62,12 +62,17 @@
         "display_name": null,
         "datatype": "string",
         "expression": {
-          "location_id_expression": {
-            "datatype": null,
-            "type": "property_name",
-            "property_name": "owner_id"
+          "location_id": {
+            "expression": {
+              "datatype": null,
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "root_doc"
```

Comment:
this doesn't need to be a root doc expression, same below

Reply:
I see a number of places where there are root doc expressions that aren't needed. Perhaps not worth changing for now.

Reply:
Yeah, going to leave it for now